### PR TITLE
create products components

### PR DESCRIPTION
**Pull request by Ahmad-Omar**

### CODE REVIEW

**Review by umarsalah — APPROVED** [review]

**Review by ameera-abed — APPROVED** [review]

**Review by Raghad-Mezied — APPROVED** [review]

**Review by Raghad-Mezied — DISMISSED** [review]

**Review by umarsalah — APPROVED** [review]

File: client/package.json [hunk 1]
```diff
@@ -4,6 +4,9 @@
   "private": true,
   "proxy": "http://localhost:3000/",
   "dependencies": {
+    "@emotion/react": "^11.4.1",
+    "@emotion/styled": "^11.3.0",
```
Comment (LinaYahya) [inline client/package.json:8]: packages already listed in package json

File: client/src/components/ProductsContainer/index.jsx [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+                key={index}
+                productImage={product.img}
+                productName={product.name}
+                productPrice={product.price}
+              />
+            );
+          }
+        })}
+      </section>
+      <Pagination
```
Comment (LinaYahya) [inline client/src/components/ProductsContainer/index.jsx:28]: maybe you need to render it conditionally when products more than six,
also, try to list 15 products, see what will happen?!

File: client/src/components/ProductsContainer/ProductCard/index.jsx [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+import "./style.css";
+function ProductCard({ productImage, productName, productPrice }) {
+  return (
+    <Card variant="outlined">
+      <CardMedia
+        component="img"
+        height="320"
+        image={productImage}
+        alt="sss"
```
Comment (LinaYahya) [inline client/src/components/ProductsContainer/ProductCard/index.jsx:16]: `alt` should be descriptive

**Review by LinaYahya — CHANGES_REQUESTED** [review]
Good work Ahmad, 
Try to fix eslint errors, and validate your props!

File: client/package.json (hunk 1 above)
Comment (Ahmad-Omar) [reply client/package.json:8]: I don't see them!

**Review by Raghad-Mezied — DISMISSED** [review]

**Review by umarsalah — APPROVED** [review]

**Review by Raghad-Mezied — APPROVED** [review]

**Review by umarsalah — APPROVED** [review]

**Review by ameera-abed — APPROVED** [review]

**Review by LinaYahya — APPROVED** [review]

**Pull request merged**